### PR TITLE
Log update vital feature modified 

### LINKED ISSUE #7290: Log update- vitals
**Is your feature request related to a problem? Please describe.**
In Log updates (Critical round), Vitals section, there is a section to mark details of pain.

Against a region, user can mark scale of pain from 0-5. 

Screenshot 2024-02-27 at 3 35 23 PM

Here the text "Low" , "Normal", "High" comes written next to the number.

**Describe the solution you'd like**

It is not useful to tag pain as "Normal"

Only retain "Low" only at 0 and "High" at 5.

- [ ] remove the appearance of "normal" against 1-4 painscale

### PR DESCRIPTION
**Pull request by saloni0419**
## Proposed Changes
In Log updates (Critical round), Vitals section, there is a section to mark details of pain.
Against a region, user can mark scale of pain from 0-5.
Here the text "Low" , "Normal", "High" comes written next to the number.

Here is the Screenshot of the fix
image

- Fixes #7290 
-  Here removed the text Normal for painscale 1-4

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [x]  removed the appearance of "normal" against 1-4 painscale

### CODE REVIEW

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

File: src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res [hunk 1]
```diff
@@ -55,13 +55,15 @@ let make = (
...
-    | (true, false) => (maxText, "#DC2626")
-    | _ => (minText, "#DC2626")
-    }
+  let getLabel = (param: float) => {
+  let minText = "Low"
+  let maxText = "High"
+  switch param {
+  | 0.0 => (minText, "#059669")
+  | 5.0 => (maxText, "#DC2626")
+  | _ => ("", "")
```
Comment (khavinshankar) [inline src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res:64]: why not have a human-readable values for all the 5 cases like,
0 -> No pain (gray)
1 -> Noticeable (yellow)
2 -> Mild (orange)
3 -> High (light red)
4 -> Extremely high (red)
5 -> Unbearable (darker red)

Comment (saloni0419) [reply src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res:64]: @khavinshankar should I change it then ?

Comment (saloni0419) [reply src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res:64]: @khavinshankar Hey I clicked on the request for review by mistake

Comment (khavinshankar) [reply src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res:64]: Yes, please do

cc: @nihal467

Comment (saloni0419) [reply src/Components/CriticalCareRecording/Pain/CriticalCare__PainInputModal.res:64]: I have opened a new pr and done the changes as you described.please review it @khavinshankar

**Comment by khavinshankar** [comment]
Closing this pr as duplicate

**Pull request closed without merging**